openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -38,8 +41,8 @@ OpenAssessment.ResponseView.prototype = {
     // before we can autosave.
     AUTO_SAVE_WAIT: 30000,
 
-    // Maximum file size (5 MB) for an attached file.
-    MAX_FILE_SIZE: 5242880,
+    // Maximum size (5 MB) for all attached files.
+    MAX_FILES_SIZE: 5242880,
```

Comment:
Double it! Per @jibsheet, we're okay to raise this to 10485760 as part of this PR.